src/styles/_card.scss

```diff
@@ -0,0 +1,79 @@
+.card {
+  width: 202px;
+  height: 410px;
```

Comment:
It's considered bad practice to have a predefined section height. It should be managed by the content itself